apps/src/lib/ui/accounts/ManageLinkedAccountsController.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ManageLinkedAccounts from './ManageLinkedAccounts';
+
+export default class ManageLinkedAccountsController {
```

Comment:
One wonders if we need this layer at all... but I guess we won't know until we've actually added behaviors.

Reply:
yeah, i'm thinking this will end up managing the overall state (which providers you're connected to, what should happen when you try to connect/disconnect from each provider, etc), but that might end up not being true

Reply:
woops, i wasn't paying attention and was referring to something else ^^^